### DISCUSSION TITLE
java-go-2-how-to-work-on-github/

### DISCUSSION THREAD

**Discussion by giscus** [lihongzheshuai/lihongzheshuai.github.io#334] [General]
# java-go-2-how-to-work-on-github/

在第一步中（一起学Java(1)-新建一个Gradle管理的Java项目）我们已经完成了项目的创建并托管到了Github上。现在自然要首先解决同学们如何下载代码和进行代码同步更新和联系的问题。这就涉及到Git的fork理念和协作模式的问题。具体介绍如下。

https://www.coderli.com/java-go-2-how-to-work-on-github/

**Comment by lihongzheshuai** [comment]
补充一个更简单的方法，fork别人的仓库后，在自己本地仓库上会出现sync fork按钮，点击即可同步到最新代码。
然后，跟自己本地库就可以。
